ee/clickhouse/queries/funnels/base.py

```diff
@@ -612,10 +631,24 @@ def _get_breakdown_select_prop(self) -> str:
             elif self._filter.breakdown_type == "group":
                 # :TRICKY: We only support string breakdown for group properties
                 assert isinstance(self._filter.breakdown, str)
-                properties_field = f"group_properties_{self._filter.breakdown_group_type_index}"
-                expression, _ = get_property_string_expr(
-                    table="groups", property_name=self._filter.breakdown, var="%(breakdown)s", column=properties_field
-                )
+
+                if self._team.actor_on_events_querying_enabled:
+                    properties_field = f"group{self._filter.breakdown_group_type_index}_properties"
+                    expression, _ = get_property_string_expr(
+                        table="groups",
```

Comment:
I think we should still rename these to `events`, as in the future if we come back and toggle `allow_denormalized_props=True`, this will get borked, since we should be searching on the events table, not persons/groups table 😅

Reply:
done